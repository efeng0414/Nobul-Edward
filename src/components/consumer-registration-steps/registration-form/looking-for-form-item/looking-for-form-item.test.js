import React from "react";
import LookingForFormItem from "./index";

describe("LookingForFormItem Component", () => {
  const componentProps = {
    getFieldDecorator: fieldName => jsx => [fieldName, jsx],
    intl: global.getIntl(),
    fieldName: "lookingFor"
  };

  const componentJSX = <LookingForFormItem {...componentProps} />;

  describe("Component Snapshot", () => {
    const componentSnapshot = global.renderer.create(componentJSX).toJSON();

    it("should match stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
