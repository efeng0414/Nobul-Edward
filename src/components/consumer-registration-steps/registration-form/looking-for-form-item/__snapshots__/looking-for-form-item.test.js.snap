// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LookingForFormItem Component Component Snapshot should match stored snapshot 1`] = `
<div
  className="consumer-registration-form-item"
>
  <div
    className="ant-row ant-form-item"
  >
    <div
      className="ant-form-item-control-wrapper"
    >
      <div
        className="ant-form-item-control"
      >
        <span
          className="ant-form-item-children"
        >
          <p
            className="consumer-registration-paragraph"
          >
            Are you looking to buy or sell?
          </p>
          lookingFor
          <div
            className="ant-radio-group ant-radio-group-outline"
          >
            <label
              className="ant-radio-wrapper"
            >
              <span
                className="ant-radio"
                style={Object {}}
              >
                <input
                  checked={false}
                  className="ant-radio-input"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  type="radio"
                  value="both"
                />
                <span
                  className="ant-radio-inner"
                />
              </span>
              <span>
                Both
              </span>
            </label>
            <label
              className="ant-radio-wrapper"
            >
              <span
                className="ant-radio"
                style={Object {}}
              >
                <input
                  checked={false}
                  className="ant-radio-input"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  type="radio"
                  value="buy"
                />
                <span
                  className="ant-radio-inner"
                />
              </span>
              <span>
                Buy
              </span>
            </label>
            <label
              className="ant-radio-wrapper"
            >
              <span
                className="ant-radio"
                style={Object {}}
              >
                <input
                  checked={false}
                  className="ant-radio-input"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  type="radio"
                  value="sell"
                />
                <span
                  className="ant-radio-inner"
                />
              </span>
              <span>
                Sell
              </span>
            </label>
          </div>
        </span>
      </div>
    </div>
  </div>
</div>
`;
